content/issues/tracking-your-work-with-issues/creating-an-issue.md

```diff
@@ -168,8 +168,14 @@ Query parameter | Example
 `projects` | `https://github.com/octo-org/octo-repo/issues/new?title=Bug+fix&projects=octo-org/1` creates an issue with the title "Bug fix" and adds it to the organization's project board 1.
 `template` | `https://github.com/octo-org/octo-repo/issues/new?template=issue_template.md` creates an issue with a template in the issue body. The `template` query parameter works with templates stored in an `ISSUE_TEMPLATE` subdirectory within the root, `docs/` or `.github/` directory in a repository. For more information, see "[Using templates to encourage useful issues and pull requests](/communities/using-templates-to-encourage-useful-issues-and-pull-requests)."
 
+{% tip %}
+
+**Tip:** You can also target custom fields in your issue templates using URL query parameters.
+
+{% endtip %}
+
```

Comment:
```suggestion

```
Oops looks like the tip somehow didn't get removed, maybe my suggestion was formatted incorrectly! Going to suggest it again and go ahead and apply the change!